lc2fen/fen.py

```diff
@@ -1,17 +1,18 @@
-"""
-FEN notation transformations.
-"""
+"""This module is responsible for FEN-related transformations."""
+
 
 PIECE_TYPES = ["r", "n", "b", "q", "k", "p", "P", "R", "N", "B", "Q", "K", "_"]
 
 
-def fen_to_board(fen):
-    """
-    Translates fen string containing only the positions of the pieces to
-    a board matrix. Empty squares are replaced by '_'.
+def fen_to_board(fen: str) -> list[list[str]]:
+    """Translate an FEN string to a board matrix.
```

Comment:
Haha, this could blow your mind, but this is one of those exceptions in English. Just because a word does not start with a vowel letter does not mean it does not have a vowel sound. For example, we say "an hour" instead of "a hour" even though "h" is not a vowel letter (the vowel letters are "a," "e," "i," "o," and "u"). What determines whether "an" or "a" is called for is whether the word starts with a vowel sound

Reply:
I know about this exception ;) However, I believe FEN is pronounced as a single word (instead of letter by letter) and in this case it would be a FEN instead of an F E N. This is supported both by wikipedia (https://en.wikipedia.org/wiki/Forsyth%E2%80%93Edwards_Notation#Definition) and the original PGN standard (Section 9.7.1 of https://ia802908.us.archive.org/26/items/pgn-standard-1994-03-12/PGN_standard_1994-03-12.txt)

Reply:
Every article I saw online used "a," a player from my chess club also uses "a," and all creators for FEN-related videos that I came across also used "a," so I gave in and changed all instances to "a!"